### ISSUE TITLE
Steiger config isn't applied in Windows platform

### ISSUE THREAD

**Issue by mx-ryabov** [feature-sliced/steiger#132]
Hey guys! 
In my side project (Next 15) I decided to try FSD and of course your great linter, but I faced with an issue that on my Windows 10 machine the steiger doesn't see my steiger.config.ts. 
At the same time I follow all the rules described in your documentation:
- the config itself in the root
- it has the following content:
```
import { defineConfig } from "steiger";
import fsd from "@feature-sliced/steiger-plugin";

const config = defineConfig([
	...fsd.configs.recommended,
	{
		ignores: ["**/*.test.tsx"],
	},
	{
		files: ["./_src/shared/**"],
		rules: {
			"fsd/no-public-api-sidestep": "off",
		},
	},
]);

export default config;
```
- the directory with FSD structure (as it's clear from the config code) is in ./_src folder.

I investigated it and found out that the current implementation considers only unix platform paths. I checked my theory running WSL and it seems so. 
Also I have a solution that seems working for both WSL and Windows platforms. What do you think about it? Let me know if I miss something.

```
function convertRelativeGlobsToAbsolute(rootPath: string, globs: Array<string>) {
  const needsConversion = (glob: string) => !isAbsolute(glob)

  return globs.map((originalGlob) => {
    const globClearPath = getGlobPath(originalGlob)

    return needsConversion(globClearPath) ? path.resolve(rootPath, originalGlob).replace(/\\/g, '/') : originalGlob
  })
}
```
In this solution I removed composeAbsolutePath function from original convertRelativeGlobsToAbsolute's implementation (since it's too unix specific) and replaced its call `replaceGlobPath(originalGlob, composeAbsolutePath(rootPath, globClearPath))` with `path.resolve(rootPath, originalGlob).replace(/\\/g, '/')`

If it's fine for you I can make a PR (or whatever way is more convenient for you, because I've never had a chance to contribute to somewhere yet)

p.s. initially I didn't understand why we need to use '/' instead of platform specific separators but it seems like minimatch requires it based on their doc. So I hope it's more or less an optimal solution

**Comment by illright** [comment]
Hi! Indeed, we have some issues with path handling on Windows. Your solution sounds great, please make a PR! Refer to [our Contributing guide](https://github.com/feature-sliced/steiger/blob/master/CONTRIBUTING.md) for help with getting started and feel free to reach out to me with any questions :)

**Comment by illright** [comment]
Closed in #134
